definitions/npm/react-with-separator_v1.x.x/flow_v0.72.x-/react-with-separator_v1.x.x.js

```diff
@@ -0,0 +1,26 @@
+type WithSeparatorProps = {
+  children?: React$Node,
+  leading?: boolean,
+  separator: string | React$Element<any>,
+  trailing?: boolean
+};
+
+type WithSeparator = React$StatelessFunctionalComponent<WithSeparatorProps>;
+
+declare module "react-with-separator" {
+  declare module.exports: WithSeparator;
+}
+
+/**
+ * We include stubs for each file inside this npm package in case you need to
+ * require those files directly. Feel free to delete any files that aren't
+ * needed.
+ */
+
+// Filename aliases
+declare module "react-with-separator/index" {
+  declare module.exports: $Exports<"react-with-separator">;
+}
+declare module "react-with-separator/index.js" {
+  declare module.exports: $Exports<"react-with-separator">;
+}
```

Comment:
I don't think we do this normally, can you talk about a use case for defining it for `index` and `index.js`? @villesau @GAntoine have you seen this sort of thing before?

Reply:
This is auto-generated by flow-typed when it creates the stub.  I can remove it.

Reply:
Ah, cool, do that then. We don't normally include those stubs.